### DISCUSSION TITLE
How to get Download link?

### DISCUSSION THREAD

**Discussion by jiungbaek** [gitpod-io/openvscode-server#374] [Q&A]
Hi There.

I want to get file Download Link like below image.(right clink at file in explorer

![image](https://user-images.githubusercontent.com/7424183/176578474-6a974c2c-84a4-459d-a8b4-c5565ff02b0b.png)

How do i use like this?

Do you guys know any extentions or how to do it?

`Download` is download to my local pc, `copy download lilnk` can download in another server.

if i can get download link like "http://test.xxx.com/testfile", i can use in server like `curl -u ID:PW http://test.xxx.com/testfile > testfile`

Thank you

**Comment by filiptronicek** [comment]
Hi @jiungbaek 👋. I took a look and this does not seem to be built in to VS Code, although it seems to be a part of the [GitLens extension](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens), even if there is a slight difference in the action name. Let me know if that works for you.

![image](https://user-images.githubusercontent.com/29888641/176625218-5ed53fc7-2a83-42c5-9c1c-49a9a25be739.png)

**Comment by jiungbaek** [comment]
Thanks, but it doesn't work for me.
`Copy Remote File Url` only get git remote file url like this`https://github.com/gitkraken/vscode-gitlens/blob/2b6fa71de3dd1884281b7ed3560a997693eeb690/LICENSE`

**Reply by filiptronicek** [reply]
Sorry @jiungbaek for the false lead. I am unsure what this extension is 😕.

**Reply by jeanp413** [reply]
If the file is in github you can do something [like this](https://github.com/gitpod-io/openvscode-server/blob/48b2ed78e83f66b426ac6ecd3556bba17e166262/extensions/github/src/links.ts#L46) but use `https://raw.githubusercontent.com/` as host prefix
